<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(167, 62, 231, 255), stop:1 rgba(0, 235, 255, 255));
</string>
  </property>
  <widget class="QPushButton" name="signin_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>407</y>
     <width>348</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Play</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(72, 49, 157);
	border-radius: 20px;
	color: rgb(255, 255, 255);

}

QPushButton:hover {
	background-color: rgba(102, 64, 191, 169);
	border-radius: 20px;
	color: rgb(255, 255, 255);
	
}

</string>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>172</y>
     <width>348</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0.2px solid #E8E7FF;
background-color: rgb(232, 231, 255);
color: #E8E7FF;
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="signin_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Play</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
color: rgb(29, 35, 77);</string>
   </property>
   <property name="text">
    <string>Имя пользователя</string>
   </property>
  </widget>
  <widget class="QLabel" name="LOGO">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>502</y>
     <width>276</width>
     <height>97</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Play</family>
     <pointsize>90</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(72, 49, 157);
background-color: qlineargradient(0);</string>
   </property>
   <property name="text">
    <string>HOLI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>87</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
border: none;
}
QPushButton:hover{
	border-bottom: 1px solid rgb(72, 49, 157);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../vks/static/svg/arrow.svg</normaloff>../../../vks/static/svg/arrow.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="password_mask">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>344</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
border: none;
}

QPushButton:hover {
	border-bottom: 1px solid rgb(72, 49, 157);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../vks/static/svg/password_mask_icon.svg</normaloff>../../../vks/static/svg/password_mask_icon.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="signin_input_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>273</y>
     <width>348</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
border: 0.5px solid #000000;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="signin_label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>252</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Play</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
color: rgb(29, 35, 77);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>339</y>
     <width>348</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
border: 0.5px solid #000000;
border-radius: 10px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_create">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>108</y>
     <width>354</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Play</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));

color: rgb(29, 35, 77);</string>
   </property>
   <property name="text">
    <string>Создать аккаунт</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>319</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Play</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
color: rgb(29, 35, 77);</string>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signin_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>348</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));
border: 0.5px solid #000000;
border-radius: 10px;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
